packages/element-api/src/services/sidetree.js

```diff
@@ -1,17 +1,37 @@
 const element = require('@transmute/element-lib');
-const ElementFirestoreAdapter = require('./element-adapter-firestore');
-const ElementMemoryAdapter = require('./element-adapter-memory');
 const { firestore } = require('./firebase');
+const ElementFirestoreAdapter = require('./element-adapter-firestore');
+
 const { getBaseConfig } = require('../config');
 
 const config = getBaseConfig();
 
+let storage;
 let db;
 
 if (process.env.NODE_ENV === 'testing') {
-  db = new ElementMemoryAdapter();
+  db = new element.adapters.database.ElementRXDBAdapter({
+    name: 'element-did.rxdb.api',
+    remote: config.couchdbRemote,
```

Comment:
I don't think we want to rely on a remote db when running tests. Tests should be without side effects

Reply:
currently, unless you specifically call awaitableSync, the DB is in memory only, so this change is not really doing anything other than using the RXDB interface that is published by element.

I agree we would not want to actually test remote db sync in every test, but we would want to test it at some point somewhere.

Reply:
See https://github.com/decentralized-identity/element/issues/90